pageserver/src/tenant/remote_timeline_client/manifest.rs

```diff
@@ -35,12 +45,21 @@ pub struct OffloadedTimelineManifest {
     pub archived_at: NaiveDateTime,
 }
 
-pub const LATEST_TENANT_MANIFEST_VERSION: usize = 1;
+/// The version number is incremented on changes, even non-breaking ones. We do not use
```

Comment:
Before we release this, let's grab a compat snapshot for posterity and add it to the historic formats test (https://docs.neon.build/storage/compat.html#historic-compatibility-testing-pageserver)

Although... it occurs to me that our compat tarball generating code might not touch anything that actually writes a manifest :-/  we should fix that (outside this PR) if so

Reply:
I wrote up https://github.com/neondatabase/neon/issues/11271 to revamp the manifest lifecycle, will address the tenant manifest upload there. Leave this PR open until I do that, but it's going to be a few days until I get to it.

Reply:
Had to tweak the snapshot a bit: https://github.com/neondatabase/neon/pull/11495